python/ray/cluster_utils.py

```diff
@@ -96,6 +97,8 @@ def __init__(self,
             shutdown_at_exit (bool): If True, registers an exit hook
                 for shutting down all started processes.
         """
+        if cluster_not_supported:
+            raise RuntimeError("cannot use Cluster on this platform")
```

Comment:
Ok, given what @carlogrisetti is saying, let's make this a logger.warning with a message like
```
Ray cluster mode is currently experimental and untested on Windows. If you are using it and running into
issues please file a report at https://github.com/ray-project/ray/issues.
```

And still skip the tests like in the rest of the PR.

Reply:
done